src/components/PagesComponent/HumanizingHealthcare/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { HumanizingHealthcareWrapper } from './style'
+import Slider from '../../UiComponent/Slider'
+import { HumanizingHealthcareSliderData } from '../../../constants'
+
+const HumanizingHealthcare = () => {
+  
+  return (
+    <HumanizingHealthcareWrapper>
```

Comment:
no need to add slider in separate component

Reply:
need to add margin padding thats why i am using in seprate component